index.js

```diff
@@ -37,7 +37,12 @@ const withCompileNodeModules = (userOptions = {}) => (nextConfig = {}) => ({
         const jsRuleNodeModules = copyJsRule(jsRule, userOptions);
 
         config.module.rules.splice(jsRuleIndex + 1, 0, jsRuleNodeModules);
-        delete config.externals;
+
+        if (options.isServer) {
+            // This is needed since Next.js requires the React to be the same instance in every page
+            // Otherwise, React would be injected individually in every page
```

Comment:
Maybe worth mention here that react hooks won’t work because of this, copy pasting the error message for reference.

Moreover I would include a link to were this regex was copied from.

Reply:
Done ✅